cfg/rippled-example.cfg

```diff
@@ -892,6 +939,12 @@
 #   [import_db]     Settings for performing a one-time import (optional)
 #   [database_path]   Path to the book-keeping databases.
 #
+#   There are 4 or 5 bookkeeping SQLite databases that the server creates and
```

Comment:
Why is it "4 or 5"? Is one of them only added with certain configurations? Without more context, the phrase "4 or 5" feels uncertain and vague. Maybe "4 to 5" would carry the right implication.

Reply:
I think `state.db` is created if you're using `online_delete`, so that would be the 5th one. I'll make the change.